FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v03/run_digi_reco.py

```diff
@@ -113,11 +113,11 @@
 
 
 # Event counter
-from Configurables import EventCounter
-eventCounter = EventCounter("EventCounter",
-                            OutputLevel=INFO,
-                            Frequency=10)
-TopAlg += [eventCounter]
+#from Configurables import EventCounter
+#eventCounter = EventCounter("EventCounter",
+#                            OutputLevel=INFO,
+#                            Frequency=10)
+#TopAlg += [eventCounter]
```

Comment:
why is this commented out?

Reply:
otherwise I am getting the following error:
ImportError: cannot import name 'EventCounter' from 'Gaudi.Configurables' (unknown location)

Reply:
Are you using the nightly? 
This was fixed in October in https://github.com/key4hep/k4FWCore/pull/252

Reply:
ah, right, I was not using the nightly. 
I will fix that.